app/content/blog/building-beyond-beginner-guitar.md

```diff
@@ -0,0 +1,248 @@
+
+## Service-specific micro hosting provider
+
+Another option I can pursue today attacks the "If everybody was on Urbit, we can make `this` thing, awesome!" problem by, well, onboarding more people on to their own instance of Urbit OS. I could have a dozen of ships running and ready to go. Prebooted, hawk installed, guitar course imported, group membership accepted, and roles assigned.
+
+We could still do the same Stripe payment, or integrate crypto payments more deeply. But the product they are buying is actually a hosted ship. I would essentially act as a small host provider for a specific service, the guitar course. Urbit being a computer in itself is incidental, but present and extensible. 
+
+Instead of purchasing access to a secret URL, purchasing the guitar course grants access to a one of these Urbit ships that has the guitar course pre-installed. 
+
+I could do this with a moon, if it seems like users aren't ready for full self-sovereignty, or I could sell them a planet from my star. Perhaps branching between the two based on if they used a Stripe or a crypto payment method. And even with the moon-based users, I could assign them a moon that was connected to the equivalent planet, earmarked from my star's children, e.g.`~sampel-palnet-dozzod-forryx` could be assigned to a credit card user, and when they are ready to upgrade then `~forryx` could issue `~sampel-palnet` to a self-custody wallet. And all this could be coordinated on Urbit via the moon, rather than depending on email.
+
+We would still need emails for marketing purposes and making sales. But once that's done, and they're on Urbit now I can communicate with them through the Moon. I can do fun things like, announce to the group:
+
+> Hey, we're going to meet on `%radio` and watch one of the lessons together to talk about it.
+
+They'll get that message in the group. And I will have already installed `%radio` on all of the hosted moons. I'll be able to send them links that I know will work, once they click on it. And they will have a way to offboard into their own self-managed solution should they be ready for the responsibility.
+
+# What's next, actually?
+
+There are a variety of decisions to make here: what kind of identities my students most want, how crypto-savvy they want to be, or if and how to get them a running instance of Urbit OS. But what is great about building this as a minimally viable Urbit-powered business is that now I have totally power of choice over making that happen. And so really the immediate next step is to keep teaching guitar and getting feedback from my students to find out what *they need*, and then be able to deliver on that without needing to hope that some SaaS service will find it profitable or will add it to their roadmap.
```

Comment:
There are a variety of decisions to make here: what kind of identities my students most want, how crypto-savvy they want to be, or if and how to get them a running instance of Urbit OS. But what is great about building this as an Urbit-powered business is that I now have total power of choice over making those decisions. And so really the immediate next step is to keep teaching guitar and getting feedback from my students to find out what *they need*, and then be able to deliver on that without needing to hope that some SaaS service will find it profitable or will add it to their roadmap.

- total power of choice to make those decisions.